integrations/jenkins/documentation.yaml

```diff
@@ -0,0 +1,77 @@
+exporter_type: included
+app_name_short: Jenkins
+app_name: {{app_name_short}}
+app_site_name: Jenkins
+app_site_url: https://www.jenkins.io/
+exporter_name: the Jenkins exporter
+exporter_pkg_name: jenkins
+exporter_repo_url: https://github.com/jenkinsci/prometheus-plugin
+additional_prereq_info: |
+  {{exporter_name}} exposes Prometheus-format metrics when the [metrics](https://plugins.jenkins.io/metrics/)
```

Comment:
Let's make this {{app_name_short}}.

We shouldn't use exporter_name at the start of the sentence because this will get expanded to "the Jenkins exporter" with uncapitalized the at the start of a sentence.

Reply:
Let's also add `{:class=external}` to these two links

Reply:
Concur!